app/views/claim_mailer/submitted.text.erb

```diff
@@ -1,23 +1,17 @@
 Dear <%= @claim.full_name %>,
 
-We‘ve received your claim to get back the student loan repayments you made through your wages at <%= @claim.eligibility.claim_school_name %> between 6 April 2018 and 5 April 2019.
+We've received your application to claim back your student loan repayments for the time you spent at <%= @claim.eligibility.claim_school.name %>.
 
-^ Your reference number is <%= @claim.reference %>.
+Your unique reference is <%= @claim.reference %>. You will need this if you contact us about your claim.
```

Comment:
Should we maybe make the reference stand out a bit (through bold text or similar)? Will Notify let us do that?

Reply:
If you add in the ^ again it will indent the text with a grey line down the side.
<img width="729" alt="Screenshot 2019-08-21 at 13 15 29" src="https://user-images.githubusercontent.com/13239597/63430990-c762a980-c415-11e9-9aef-0ac683fb6ab4.png">